Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -1253,6 +1434,13 @@ void vtkMRMLSliceNode::PrintSelf(ostream& os, vtkIndent indent)
   os << indent << "SliceToRAS: \n";
   this->SliceToRAS->PrintSelf(os, indent.GetNextIndent());
 
+  std::map<vtkStdString, vtkMatrix4x4*>::iterator it;
+  for (it = this->OrientationMatrices.begin(); it != this->OrientationMatrices.end(); ++it)
+    {
+    os << indent << "OrientationMatrix"<< it->first <<": \n";
```

Comment:
If you want to serialize like this then you have to make sure to reject names that contain space or special characters. Instead, it would be better to serialize all names and matrices into a single element. Name could be encoded/decoded to handle special characters (URLEncodeString/URLDecodeString).